.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: generic
```

Comment:
Using Travis here is quick but deviates from the standards on GOV.UK. Either make this a Jenkins check (:sob:, no good for content designers because of GitHub auth + needs VPN), or we can pave a new way and I can whip up a GitHub Action?

Reply:
There's already a tiny amount of prior art for Actions, at least, and it works well. I'll do this tomorrow and we can compare and contrast!

Reply:
There's only ten seconds in it on build times at this time of night, but Actions wins. 😛

Reply:
Every second counts!

I've removed Travis.